xbmc/interfaces/legacy/mod_python/Makefile

```diff
@@ -8,6 +8,8 @@ LIB=legacy-mod_python.a
 
 CXXFLAGS+= -DSWIGRUNTIME_DEBUG -DSTATIC_LINKED
 
+INCLUDES += -I/usr/include/python2.7
```

Comment:
This doesn't work with depends/cross builds. We probably have to turn this `Makefile` into a `Makefile.in` and use
```
INCLUDES += @PYTHON_CPPFLAGS@
```